base/.eslintrc.json

```diff
@@ -26,7 +26,6 @@
     "no-self-compare": 2,
     "no-useless-call": 2,
     "no-shadow-restricted-names": 2,
-    "no-shadow": 2,
```

Comment:
Put this back.

Reply:
https://github.com/dabapps/eslint-config-dabapps/pull/75#discussion_r592247539

Reply:
Did you mean to duplicate the rule but in the case file keep it as `"no-shadow": 0,`?

Reply:
This error only occurs in typescript files, so you only need to adjust the rules in the typescript overrides. Instead you have removed the rule from our base config which means that none of our javascript files will be linted for this rule.

Reply:
👍